src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IepLevelResourceTest.kt

```diff
@@ -42,6 +42,18 @@ class IepLevelResourceTest : SqsIntegrationTestBase() {
       .isUnauthorized
   }
 
+  @Test
+  fun `Prison API '404 Not Found' responses are handled instead of responding 500 Internal Server Error`() {
+    val bookingId: Long = 1234134
+
+    prisonApiMockServer.stubApi404for("/api/bookings/$bookingId/iepSummary?withDetails=true")
+
+    webTestClient.get().uri("/iep/reviews/booking/$bookingId")
```

Comment:
You could pass `use-nomis-data=true` so it's a bit clearer we're making a call to PrisonAPI?

Reply:
True, how wonder why the test works tho 🤔 - I'll change and also have a play

Reply:
[By default](https://github.com/ministryofjustice/hmpps-incentives-api/blob/aa445fea20258a2b7c8776e927e362a0b3749f5a/src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IepLevelResource.kt#L107) `use-nomis-data` is set to true - I just thought we could include it in the test for clarity but not a big issue 😄

Reply:
yeah, just realised it was the default (as it makes sense) but it's better to be explicit 👍

Reply:
https://github.com/ministryofjustice/hmpps-incentives-api/pull/154/commits/f0431201be252ad871fc22d010d47fb036d33160